apis/kubedb/v1/postgres_helpers.go

```diff
@@ -139,6 +143,18 @@ func (p Postgres) GoverningServiceName() string {
 	return meta_util.NameWithSuffix(p.ServiceName(), "pods")
 }
 
+func (p Postgres) OffshootDistributedRBACName() string {
+	return meta_util.NameWithSuffix(p.OffshootName(), "rbac")
+}
+
+func (p Postgres) OffshootDistributedServiceExportName() string {
+	return meta_util.NameWithSuffix(p.OffshootName(), "serviceexport")
+}
+
+func (p Postgres) OffshootDistributedAuthSecretName() string {
+	return meta_util.NameWithSuffix(p.OffshootName(), "auth")
```

Comment:
DistributedCustomConfigSecretNameSuffix = "custom-config"
	DistributedRBACNameSuffix               = "rbac"
	DistributedServiceExportNameSuffix      = "serviceexports"
	
	And Remove the similar mariadb consts